workflow/react-project/src/components/common/PasswordInput/index.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+import { faEyeSlash } from '@fortawesome/free-regular-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import './style.less';
+
+const PasswordInput = ({ title, errorsMessage, ...inputProps }) => {
+  const [isPasswordVisible, setPasswordVisible] = useState(false);
```

Comment:
not necessary write so long names, it's should be clear to understand:
for ex here, we are inside `PasswordInput`, so we don't need use word `password` everywhere 

`isPasswordVisible` can be renamed to `isValueVisible` (it controls values visibility)